src/api/spec/support/features/custom_forms_helper.rb

```diff
@@ -0,0 +1,11 @@
+module CustomFormsHelper
```

Comment:
It's good to have a workaround, but I believe it's unneeded. Even though custom Bootstrap checkboxes are not directly clickable, the label is.

The `check/uncheck` method have [the option `allow_label_click`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FActions:check) which works with Bootstrap and non-Bootstrap checkboxes.

One example from one of our feature specs:

https://github.com/openSUSE/open-build-service/blob/2b5a83a9e27dbfc353b23df0cb35c4ad105fe73c/src/api/spec/features/webui/projects_spec.rb#L139-L140

Reply:
:thumbsup: Nice. I dropped the commit.